src/hooks/useInterval.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react'
+
+export const useInterval = (cb: () => void, delay: number) => {
+  useEffect(() => {
+    const id = setInterval(cb, delay)
+
+    return () => {
+      clearInterval(id)
+    }
```

Comment:
this gets called on every change
```
console.logs
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
Web3Provider.tsx:262 gasPrice _BigNumber {_hex: '0x174876e800', _isBigNumber: true}
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
useInterval.tsx:9 clearInterval
useInterval.tsx:5 setInterval
```

Reply:
Yeah, you are right, this doesn't work as expected. Fixed.

Reply:
still getting:
```
3:28:18 PM [vite] hmr invalidate /src/providers/Web3Provider.tsx Could not Fast Refresh. Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
3:28:18 PM [vite] hmr update /src/App.tsx, /src/components/Layout/index.tsx, /src/pages/ConnectWallet/index.tsx, /src/providers/WrapFormProvider.tsx, /src/pages/Wrapper/index.tsx, /src/pages/Transaction/index.tsx, /src/components/Account/index.tsx
3:28:19 PM [vite] hmr invalidate /src/providers/WrapFormProvider.tsx Could not Fast Refresh. Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
```

Reply:
Exported `createContext` calls seems to be causing this. Thats lame, I moved `useWrapForm`(because it was exported) out of `WrapFormProvider.tsx`, but now `WrapFormContext` needs to be exported to be used in `useWrapForm`. And for the Fast Refresh to work, `.tsx` files need to export exactly one Component.